db/fixtures/lessons/html_and_css_lessons.rb

```diff
@@ -217,5 +217,61 @@ def html_and_css_lessons
       url: '/html_css/conclusion.md',
       identifier_uuid: '58c50ae7-76a9-443d-b6df-227ec3e8b11b',
     },
+    'Introduction to Web Accessibility' => {
+      title: 'Introduction to Web Accessibility',
+      description: 'Learn what Accessibility means to you as a web developer',
```

Comment:
Just a nitpick but are we using Accessibility as a proper noun here?

Reply:
no lol weird typo